test-suite/src/test/require-jdk8/org/scalajs/testsuite/javalib/time/temporal/ChronoUnitTest.scala

```diff
@@ -8,6 +8,49 @@ object ChronoUnitTest extends JasmineTest {
   import ChronoUnit._
 
   describe("java.time.temporal.ChronoUnit") {
+    it("should respond to `isDurationEstimated`") {
+      for (u <- ChronoUnit.values())
+        expect(u.isDurationEstimated != u.isTimeBased).toBeTruthy
+    }
```

Comment:
@nicolasstucki The test for `isDurationEstimated` is here. `isDurationEstimated` always return the opposite of `isTimeBased`.